RecoLocalTracker/SiPhase2Clusterizer/test/ClustersValidation.cc

```diff
@@ -33,6 +33,8 @@
 #include "SimDataFormats/Vertex/interface/SimVertexContainer.h"
 #include "SimDataFormats/TrackingHit/interface/PSimHitContainer.h"
 
+#include "SimTracker/SiPhase2Digitizer/interface/Phase2TrackerDigiCommon.h"
```

Comment:
Please move non-sim code to some other place.
Reco\* subsystem should depend on the Sim minimally.

It's OK for validation code, but for the Phase2TrackerDigiCommon it doesn't seem justifiable, since the code just deals with TrackerTopology and DetId
SimTracker/SiPhase2Digitizer/src/Phase2TrackerDigiCommon.cc